geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementService.java

```diff
@@ -35,6 +35,7 @@
    * @param config this holds the configuration attributes of the element to be created on the
    *        cluster, as well as the group this config belongs to
    * @return a {@link ClusterManagementRealizationResult} indicating the success of the creation
+   * @throws RuntimeException if unsuccessful
```

Comment:
It does not seem right to throw the very general "RuntimeException". Instead a more specific subclass of RuntimeException should be part of our external API and that is what we would say we throw.
Note it does not need to be a checked exception. If it is a checked exception then all users of the api have to handle that exception. On some of the methods on this interface we throw other runtime exceptions (i.e. IllegalArgumentException, NoMemberException, EntityExistsException). So if we have some other reason for failing would should probably just have another, subclass of either RuntimeException or GemFireException. We should also include in the javadocs an explanation for why the exception will be thrown

Reply:
done